rpm/libiphb.spec

```diff
@@ -84,8 +61,6 @@ rm -rf %{buildroot}
 %defattr(-,root,root,-)
 %{_libdir}/libiphb.so.*
 %doc COPYING AUTHORS INSTALL README NEWS ChangeLog
```

Comment:
We should probably move most of these to -devel only, no need to have these on normal system.